crates/biome_html_formatter/tests/specs/prettier/angular/typeof-expression/typeof-expression.html

```
@@ -0,0 +1,36 @@
+<div
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)